span-normalizer/build.gradle.kts

```diff
@@ -47,4 +47,5 @@ dependencies {
   runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.13.3")
 
   testImplementation("org.junit.jupiter:junit-jupiter:5.6.2")
+  testImplementation("org.mockito:mockito-core:2.19.0")
```

Comment:
I think 3.3.3 is the latest version of this.

Reply:
Removing it, not being used